Library/Config.ts

```diff
@@ -46,6 +46,11 @@ class Config {
     public httpAgent: http.Agent;
     /** An https.Agent to use for SDK HTTPS traffic (Optional, Default undefined) */
     public httpsAgent: https.Agent;
+    /** (Off by default) If true, when depdency telemetry is autocollected, more information about it
```

Comment:
Specify this only works with MongoDB, we can update later if MySQL and PostgreSQL support is added